### PR TITLE
Enhancements & Fixes & Renames [after #3318] plus get live Run Time info and a bit PEP8 🌶

### PR DESCRIPTION
**Pull request by uluQulu**
## Hi all

I wrote commit names & short descriptions quite explanative, so I think, this little PR will not deserve a big description.

One thing is,
I did what I talked at https://github.com/timgrossmann/InstaPy/pull/3318#issuecomment-441328629;
Now, replying to commens has its own controller setting - `set_do_reply_to_comments`, after all, it is no more dependant to `set_do_comment`.

Currently, replying to comments are being done as normal comments by just mentioning commenter's username, but, I think, some day the web service will also have the same functionality as in the mobile app. That's why it's one another reason for replies to comments must be handled separately.

Also, renamed `set_reply_comments` to `set_comment_replies` due to several reasons.
If you wonder, ask out.
Such as, replies can be send any other places else than to comments, e.g. to stories..
So I chose the generic naming model to avoid confusion and have pretty clarity.

---

The feature of _run time_ is an idea of another InstaPy fella (_couldn't remember name or find that thread_) that I just decided to write cos that's a very useful feature, to me.

---
This PR doesn't have something big, it's more like _missing commits_ 🛸

Cheers 😁
---

### CODE REVIEW

**Comment by sionking** [comment]
You are doing so much for the community, but I think we are missing basic core design and functionalty and not more features, so I suggest we need to take the dev time for stabilty and code refactoring of Instapy, since right now it id damaged.

I have at least one chrome crash a day for each user, I think we should put some effort on web driver recovering and general flow correctness.

**Comment by w4ngel** [comment]
> You are doing so much for the community, but I think we are missing basic core design and functionalty and not more features, so I suggest we need to take the dev time for stabilty and code refactoring of Instapy, since right now it id damaged.
> 
> I have at least one chrome crash a day for each user, I think we should put some effort on web driver recovering and general flow correctness.

Chromedriver generally has a LOT of issues. Firefox headless is significantly more stable from my experience. Only issue i have had is that sometimes it's slightly slower loading elements, but our explicit wait accounts for that.

**Comment by uluQulu** [comment]
I just found out the owner of the idea of `run_time()` - @Areahints, https://github.com/timgrossmann/InstaPy/issues/3405#issue-383059343.

**Comment by timgrossmann** [comment]
@uluQulu Is this ready for merging?
Does it intervene with #3596 ?

**Comment by uluQulu** [comment]
Yes, @timgrossmann. I think so.
It's just its Changelog entry's date to be updated before the merge.

I have shared nothing related to browser abstraction yet, so, no intervention is expected.

**Comment by timgrossmann** [comment]
@uluQulu Perfect, thank you very much!
I'll update the changelog once it gets merged.

File: instapy/commenters_util.py [hunk 1]
```diff
@@ -277,17 +276,22 @@ def likers_from_photo(browser, amount=20):
         likers = []
 
         for liker in liked_this:
-            if "like this" not in liker.text:
+            if " like this" not in liker.text:
                 likers.append(liker.text)
 
         if check_exists_by_xpath(browser, liked_counter_button):
-            if re.match(r'\d+ others', liked_this[-1].text):
```
Comment (timgrossmann) [inline instapy/commenters_util.py:284]: I think this was introduced in a PR earlier.
Could you please rebase to the latest master so we are not overriding anything "new"?

Comment (uluQulu) [reply instapy/commenters_util.py:284]: Have you read https://github.com/timgrossmann/InstaPy/issues/3573#issuecomment-447570792?
BTW files are up-to-date (_see 24acb17efad8d662df65099330704491df4a6189_).

Comment (timgrossmann) [reply instapy/commenters_util.py:284]: Correct me if I'm wrong, please.
You're basically saying that the regex usage here is not necessary and your fix is working fine like this? => Perfect

Comment (uluQulu) [reply instapy/commenters_util.py:284]: Yes, @timgrossmann 
The regular expression used in there tends to match a digit followed by `" others"` pattern.
And a comma inside digits makes it misbehave.

I have seen that element [inside DOM] before suggesting a solution and it is enough to just check the `" others"` pattern cos it is obvious that there will be a digit before that string.
Alongside it, as you can see, I also renamed `"like this"` to `" like this"` and `"likes"` to `" likes"` to increase accuracy a bit.

**Comment by timgrossmann** [comment]
@uluQulu It's working as expected on my instances. 
Just to make sure I haven't missed something in the code review, this PR is only adding new functionality and not changing crucial already existing code, right? 
Looks like a lot of the changes you did were in fact the PEP8 formatting 👍

**Comment by sionking** [comment]
I have this working (I hope)

**Comment by timgrossmann** [comment]
@uluQulu @sionking Let's see if someone else can/will find an issue 😉 
If not, even better.

As always, thank you very much for this nice feature, @uluQulu !

**Comment by converge** [comment]
amazing code quality improvement, congratulations mate!! @uluQulu

**Comment by uluQulu** [comment]
### Hey all
Thanks a bunch @timgrossmann, it will be a bit late response, but,
>...this PR is only adding new functionality and not changing crucial already existing code, right?...

I understand your concern and it is the same for me, to reply to you, I checked code one more time again. The point is although my accuracy, when a PR is multi-dimensional, not small and has lots of little changes, it is difficult to differentiate 😂 And PEP 8 changes make it more difficult..

But good news is that last times all my commits are unique and it is good to differentiate.
There are 19 commits and in particular, commit of https://github.com/timgrossmann/InstaPy/pull/3460/commits/65ed02390d8ee9451eb23f4990df626b857d69d5 holds the PEP 8 changes [only].

If you asked me about it right after I freshly opened the PR, I would quickly reply but after ~3 weeks, hm, I have not so good memory 🌝 so analyzed again..
<h6>Well, yes this PR has nothing unmentioned, all the changes are as is in the commit descriptions.</h6>

---

I'm so thankful to @sionking for checking my PRs 🙋🏼‍♂️

Many thanks @converge, I like PEP 8 much more now. I did 2 big phases of compliance (_at #3318 and here_) and now, I think, only small places remain PEPless. After that, code will look more amazing 😎


---

Cheers 😁
---

**Comment by timgrossmann** [comment]
@uluQulu Absolutely understand that, thank you very much for double checking 😉 

I think once we have #3579 in place, we only have to check the PRs for usefulness and and that there is no code parts that have malicious behaviour.

Thanks agin.

**Pull request merged**